api/hub/src/datasources/moderation-decision.ts

```diff
@@ -271,8 +271,8 @@ class ModerationDecisionAPI extends DataSource {
       const moderator = finalDecision.moderator.startsWith('0x')
         ? await profileAPI.getProfile(finalDecision.moderator)
         : await profileAPI.resolveProfile(finalDecision.moderator);
-      finalDecision.moderator = moderator.pubKey;
       finalDecision = Object.assign({}, finalDecision, {
+        moderator: moderator.pubKey,
```

Comment:
I was suggesting something like 
```
Object.assign(
    finalDecision,
    { 
        moderator: moderator.pubKey,
        moderatorProfile: { ethAddress: moderator.ethAddress, pubKey: moderator.pubKey }
    }
)
```
that would replace the entire line 274

Reply:
Got it. I've pushed a fix for both comments.